octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java

```diff
@@ -193,25 +199,38 @@ private void drawEntityFields(Set<String> shownFields, EntityModelWrapper entity
             CLabel labelFieldName = new CLabel(columnComposite, SWT.NONE);
             labelFieldName.setText(fieldLabelMap.get(fieldName));
             labelFieldName.setFont(JFaceResources.getFontRegistry().getBold(JFaceResources.DEFAULT_FONT));
-            GridData labelFieldNameGridData = new GridData(SWT.LEFT, SWT.CENTER, false, false, 1, 1);
+            GridData labelFieldNameGridData = new GridData(SWT.LEFT, SWT.CENTER, false, true, 1, 1);
+            labelFieldNameGridData.heightHint = 35;
             labelFieldName.setLayoutData(labelFieldNameGridData);
 
             FieldEditor fieldEditor = fieldEditorFactory.createFieldEditor(columnComposite, entityModelWrapper, fieldName);
+
+            // Attach a clear handler to the sprint field editor
+            // TODO: full mvc
+            if (EntityFieldsConstants.FIELD_SPRINT.equals(fieldName)) {
+                entityModelWrapper.addFieldModelChangedHandler(fieldModel -> {
+                    if (EntityFieldsConstants.FIELD_RELEASE.equals(fieldModel.getName())) {
+                        entityModelWrapper.setValue(new ReferenceFieldModel(EntityFieldsConstants.FIELD_SPRINT, null));
+                        fieldEditor.setField(entityModelWrapper, EntityFieldsConstants.FIELD_SPRINT);
+                    }
+                });
+            }
+
             GridData fieldEditorGridData = new GridData(SWT.FILL, SWT.FILL, true, true, 1, 1);
-            fieldEditorGridData.heightHint = 30;
+            fieldEditorGridData.heightHint = 35;
             Control fieldEditorControl = (Control) fieldEditor;
             fieldEditorControl.setLayoutData(fieldEditorGridData);
             fieldEditorControl.setForeground(foregroundColor);
 
-            //In case of uneven number of fields, the row heights would be inconsistent
-            //and Eclipse does not know how to scale correctly
+            // In case of uneven number of fields, the row heights would be
+            // inconsistent
+            // and Eclipse does not know how to scale correctly
             if (i == shownFields.size() - 1 && i % 2 != 1) {
```

Comment:
@atothhpe @vladbenchea I know that you didn't write this, but i % 2 != 1 doesn't seem right.
Perhaps you can update this while it is still fresh in our minds to take into account the skipped fields (line 186 from above)

Reply:
hahahahaha